masonry/src/lib.rs

```diff
@@ -3,40 +3,35 @@
 
 //! A framework that aims to provide the foundation for Rust GUI libraries.
 //!
-//! Masonry gives you a platform to create windows (using Glazier as a backend) each with a tree of widgets. It also gives you tools to inspect that widget tree at runtime, write unit tests on it, and generally have an easier time debugging and maintaining your app.
+//! Masonry gives you a platform to create windows (using [winit] as a backend) each with a tree of widgets. It also gives you tools to inspect that widget tree at runtime, write unit tests on it, and generally have an easier time debugging and maintaining your app.
 //!
 //! The framework is not opinionated about what your user-facing abstraction will be: you can implement immediate-mode GUI, the Elm architecture, functional reactive GUI, etc, on top of Masonry.
 //!
-//! This project was originally was originally a fork of Druid that emerged from discussions I had with Raph Levien and Colin Rofls about what it would look like to turn Druid into a foundational library.
+//! Masonry was originally a fork of [Druid] that emerged from discussions within the Linebender community about what it would look like to turn Druid into a foundational library.
 //!
 //! ## Example
 //!
-//! **(TODO: FIX THIS EXAMPLE)**
-//! The todo-list example looks like this:
+//! The to-do-list example looks like this:
 //!
-//! ```ignore
-//! use masonry::widget::{prelude::*, TextBox};
-//! use masonry::widget::{Button, Flex, Label, Portal, WidgetMut};
-//! use masonry::Action;
-//! use masonry::{AppDelegate, AppLauncher, DelegateCtx, WindowDescription, WindowId};
+//! ```
```

Comment:
I'm not a fan of having the same example in three different places, but since doc examples aren't runnable, at least one of those duplications is inevitable. I expect we'll lose the second when we move to `#[doc = concat!(include!())]`